vispy/visuals/graphs/layouts/networkx_layout.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+from ..util import _straight_line_vertices, issparse
+import numpy as np, importlib
+
+
+class NetworkxCoordinates:
+    def __init__(self, graph, layout = None, *args, **kwargs):
+        self.graph = graph
+        self.positions = np.zeros((len(graph), 2), dtype = np.float32)
+        # default random positions
+        if type(layout) is type(None):
+            self.positions = np.random.rand(*self.positions.shape)
+
+        # check for networkx
+        elif isinstance(layout, str):
+            if USE_NETWORKX := importlib.util.find_spec("networkx"):
```

Comment:
We still support python 3.6+, the walrus operator was introduced in python 3.8. Could you rework this line?

Reply:
Maybe just a try/except at the top of the module? That's how we normally do this type of thing in the other vispy modules.

Reply:
ah my bad in my haste I haven't checked if there is a developers manual. I use networks quite a lot and have used vispy in the past for another project and was pretty impressed with the performance. I have edited the code slightly to do a try/except import. Alternatively we could use `importlib` for this find `util.find_spec` being the default way of looking for packages.

Reply:
The latest commit has
- fixed spec according to python 3.6; removed walrus operator
- added try except import
- added doc string